src/lib/audienceChannel/styles/slides.css

```diff
@@ -0,0 +1,50 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font: 13px Helvetica, Arial;
+}
+
+#present {
+  width: 800px;
+  height: 600px;
+  margin: 0 auto;
+  margin-top: 0px
+}
+
+#previous {
```

Comment:
previous button is used by both slides.html and phresenter.html for question dialog, can it be put into a common css?